lib/forwardanalyzer.cpp

```diff
@@ -524,7 +524,8 @@ struct ForwardTraversal {
                     if (!condTok->hasKnownIntValue() || inLoop) {
                         if (!analyzer->lowerToPossible())
                             return Break(Analyzer::Terminate::Bail);
-                    } else if (condTok->values().front().intvalue == static_cast<long long>(inElse)) {
+#pragma warning (suppress : 4805) // unsafe mix of types
```

Comment:
I would rather want to have a global directive. Is it possible to disable this warning in the cmake and visual studio solution instead.

Reply:
I'm not sure if this warning is always unhelpful.
Looking at compileroptions.cmake, it seems that a number of warnings are already deactivated there (why?), All warnings are active in the supplied project file.

Reply:
I have seen msvc fixes for conversions before in PRs.. would be nice to avoid that in future..

Reply:
I wasn't looking at the correct setting in the project file, the warnings are also deactivated there. Which means my latest changes are in the wrong place...
I guess I will close this PR, but it would still be nice to get release-windows working again...